test/Sema/diag_ownership_lazy.swift

```diff
@@ -1,7 +1,19 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -verify-ignore-unknown
+
+// -verify-ignore-unknown is for:
+// <unknown>:0: error: unexpected note produced: 'self' declared here
 
 class LazyReferenceClass {
-  lazy unowned(unsafe) var unsafeSelf = { self }() // expected-error{{lazy properties must not have reference attributes}}
-  lazy weak var weakSelf = self // expected-error{{lazy properties must not have reference attributes}}
-  lazy unowned var unownedSelf = self // expected-error{{lazy properties must not have reference attributes}}
+  // expected-note@-1 {{type declared here}}
+  // expected-note@-2 {{type declared here}}
```

Comment:
For future reference, you can write `expected-note@-1 2 {{type declared here}}` to mean "I expect 2 of this note 1 line up".